src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs

```diff
@@ -251,12 +251,8 @@ private static bool PlatformAnalysisAllowed(AnalyzerOptions options, Compilation
             {
                 try
                 {
-                    if (platformSpecificOperations.IsEmpty)
-                    {
-                        return;
-                    }
-
-                    if (guardMethods.IsEmpty || !(context.OperationBlocks.GetControlFlowGraph() is { } cfg))
+                    if (platformSpecificOperations.IsEmpty || guardMethods.IsEmpty ||
+                        !(context.OperationBlocks.GetControlFlowGraph() is { } cfg))
```

Comment:
Nit: either separate out each part of the condition into a separate line OR keep the entire condition on the same line.
```suggestion
                    if (platformSpecificOperations.IsEmpty ||
                        guardMethods.IsEmpty ||
                        !(context.OperationBlocks.GetControlFlowGraph() is { } cfg))
```

Reply:
Also tagging @CyrusNajmabadi - seems like a good candidate for your whitespace/newline formatting analyzers